packages/api-react/src/services/wallet.ts

```diff
@@ -222,16 +222,20 @@ export const walletApi = apiWithTag.injectEndpoints({
       invalidatesTags: [{ type: 'Wallets', id: 'LIST' }],
     }),
 
-    deleteUnconfirmedTransactions: build.mutation<any, { 
+    deleteUnconfirmedTransactions: build.mutation<any, {
       walletId: number;
     }>({
       query: ({ walletId }) => ({
         command: 'deleteUnconfirmedTransactions',
         service: Wallet,
         args: [walletId],
       }),
+      invalidatesTags: [
+        { type: 'Transactions', id: 'LIST' },
+        'TransactionCount',
```

Comment:
we should add wallet Id as id and invalidate TransactionCount just for specific Wallet.
it will be connected via providesTags where you need to add walletId as well {type: 'TransactionCount', id: walletId }